wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,167 @@
+# Rapid Monthly osu! Tournament Invitational 5
+
+The Rapid Monthly osu! Tournament Invitational \#5 is a tournament organized and run by the Rapid Tournaments team.
+It is an exclusive tournament where only players who have reached semi-finals in one of the last four RMoT's get invited.
+
+The tournament brackets can be found on the [Challonge page](https://challonge.com/rmotinvitational5). Most of the information here complies with the [forum post](/community/forums/topics/741126).
+
+![France][flag_FR] [Prosdokies](/users/3852436 "Prosdokies") won the fifth Invitational, beating ![Russian Federation][flag_RU] [Well Halen](/users/8660244 "Well Halen") in the Grand Finals
```

Comment:
for the flag images I think you should leave the alt text empty (like `![][flag_FR]`), it would be kinda odd to just see "France" written in-line if the image fails to load

ofc this happens many times later

Reply:
I guess so, flag images are used that way in other articles as well...